### PR TITLE
Seam Reports

### PR DESCRIPTION
**Pull request by cmgiven**

### CODE REVIEW

File: app/models/document.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ def self.from_vbms_document(vbms_document)
     new(
       type: TYPES[vbms_document.doc_type] || :other,
       alt_types: (vbms_document.alt_doc_types || []).map { |type| ALT_TYPES[type] },
+      vbms_doc_type: vbms_document.doc_type,
```
Comment (shanear) [inline app/models/document.rb]: This is storing the same info as `#type`. Should we try and consolidate these two values?

File: config/database.yml [hunk 2]
```diff
@@ -31,9 +31,9 @@ staging_vacols:
   adapter: oracle_enhanced
   username: <%= ENV["VACOLS_USERNAME"] %>
   password: <%= ENV["VACOLS_PASSWORD"] %>
-  host: vacols.dev.vaco.va.gov
-  port: 1526
-  database: BVAP.VACO.VA.GOV
+  host: <%= ENV["VACOLS_HOST"] || "vacols.dev.vaco.va.gov" %>
```
Comment (shanear) [inline config/database.yml]: ❤️

File: app/models/appeal.rb [hunk 3]
```diff
@@ -50,6 +52,10 @@ def representative_type
     end
   end
 
+  def hearing_pending
```
Comment (shanear) [inline app/models/appeal.rb]: would be good if this had a unit test

File: app/models/appeal.rb [hunk 4]
```diff
@@ -165,9 +171,12 @@ def from_records(case_record:, folder_record:, correspondent_record:)
         form9_date: normalize_vacols_date(case_record.bfd19),
         ssoc_dates: ssoc_dates_from(case_record),
         hearing_type: VACOLS::Case::HEARING_TYPES[case_record.bfha],
+        hearing_requested: (case_record.bfhr == "1" || case_record.bfhr == "2"),
```
Comment (shanear) [inline app/models/appeal.rb]: we should update the unit tests for these changes

File: app/models/document.rb (hunk 1 above)
Comment (shanear) [inline app/models/document.rb]: I'm not sure what you mean. `type` is a one to one mapping with `vbms_document.doc_type`. If we don't a possible value for `vbms_document.doc_type` accounted for in `Document::TYPES` then we should add it.

**Comment by cmgiven** [comment]
Unit tests and consolidation of `type` complete. Agreed on automating periodic rerunning of these reports; moved to #369.

**Comment by cmgiven** [comment]
That quickie fix worked. @shanear, back to you.

File: app/models/appeal.rb (hunk 4 above)
Comment (shanear) [inline app/models/appeal.rb]: @cmgiven we still need to update the `.from_records` spec to test for these new fields (`hearing_requested`, `hearing_held`, `merged`, etc.)

Comment (cmgiven) [inline app/models/appeal.rb]: @shanear: `hearing_requested` and `hearing_held` were already in there. Added `merged`.

**Pull request merged**